src/components/store/BearStore.ts

```diff
@@ -0,0 +1,33 @@
+import { create } from 'zustand';
+import { OrderSubmitProps } from '../content/order/create/OrderSubmit';
+import { DeployParam } from '../content/order/formElements/CommonTypes';
+
+interface OrderBearState {
```

Comment:
here we can avoid the word "bear". We can call it "OrderParametersState"
same for the file name also.